user-guide/Standard_Apps/SRM/SRM_migrating/SRM_migrating_profile_instances.md

```diff
@@ -7,7 +7,7 @@ uid: SRM_migrating_profile_instances
 To export and import profile instances, you can use the *PFM_ProfileInstancesImportExport* Automation script.
 
 > [!IMPORTANT]
-> The *PFM_ProfileInstancesImportExport* script needs to be able to generate an .xlsx file. For this reason, before you use the script, you need to make sure *AccessDatabaseEngine.exe* is installed in the DMS. You can download this from [microsoft.com](https://www.microsoft.com/en-in/download/details.aspx?id=54920).
+> The *PFM_ProfileInstancesImportExport* script needs to be able to generate an .xlsx file. For this reason, before you use the script, you need to make sure the x86 version of *AccessDatabaseEngine.exe* is installed in the DMS. You can download this from [microsoft.com](https://www.microsoft.com/en-in/download/details.aspx?id=54920).
```

Comment:
@EdithLansens could you rephrase? So to be more in line with what's on the download page.
> the 32-bit (x86) version of

Reply:
I've updated with a new commit